### PR TITLE
Task6

### PR DESCRIPTION
**Pull request by Mateen-Ahmad5**
1. Enhance the signup form
2. Use JavaScript to send an HTTP request to the API with the new user’s details. (name, email, password). 
3. Display appropriate messages based on the API response (success or error) after the form submission.

### CODE REVIEW

File: signup.css [hunk 1]
```diff
@@ -1,78 +1,121 @@
...
     width: 100%;
-    background-color: #007bff;
-    color: #fff;
-    padding: 10px 0;
+    background-color: #fff;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    padding: 20px 0;
+    position: fixed;
+    top: 0;
+    left: 0;
```
Comment (sikander-codes) [inline signup.css]: left 0 is useless here

File: signup.html [hunk 2]
```diff
@@ -1,36 +1,41 @@
...
-              <input type="password" id="password" placeholder="Enter your password" >
-              <span id="passwordError" class="error"></span>
-          </div>
-           <button type="submit">Submit</button>
-           <div id="message" class="message"></div>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="signup.css">
+    <title>HTML Form Validation</title>
```
Comment (sikander-codes) [inline signup.html]: Add a page tittle which is more relevant like Signup

File: signup.css (hunk 1 above)
Comment (Mateen-Ahmad5) [reply signup.css]: solve this

File: signup.html (hunk 2 above)
Comment (Mateen-Ahmad5) [reply signup.html]: solve this

**Review by sikander-codes — APPROVED** [review]

**Pull request merged**